replace_n_launch.sh

```diff
@@ -14,12 +14,15 @@ Darwin)
     launchctl load "/Library/LaunchDaemons/sh.webexec.daemon.plist"
     ;;
 Linux)
-    systemctl stop webexec.service
```

Comment:
Actually, most systems today **do** have systemd, and I think it is better to support it.
The major systems I know that doesn't use systemd are:
1. WSL - will probably be solved soon
2. devuan and other distros that doesn't use systemd on purpose

I think I would check for `command -v systemctl` as you did elsewhere and still use it if available, and drop to init.d only if it is unavailable.